src/main/java/it/pagopa/pn/commons/pnclients/addressbook/AddressBook.java

```diff
@@ -0,0 +1,8 @@
+package it.pagopa.pn.commons.pnclients.addressbook;
+
+import it.pagopa.pn.api.dto.addressbook.AddressBookEntry;
+
+public interface AddressBook {
+
+    AddressBookEntry getAddresses( String taxId );
```

Comment:
**?**
``taxId`` è un nome così o è li per dire _codice fiscale_?

Se è la seconda, mi raccomando a chiarire anche nel codice che non è il codice vero, ma il suo token.

Chiamalo ``tokenisedFiscalCode`` ad es. o in generale ``tokenisedXxx``.

Da applicarsi a cascata altrove se necessario.